client/plots/singleCellPlot.js

```diff
@@ -204,7 +205,9 @@ class singleCellPlot {
 						const config = {
 							chartType: 'gsea',
 							gsea_params: gsea_params,
-							//if getPlotHolder is defined use this.insertBeforeId, needed for GDC
+							// if getPlotHolder is defined, use this.mainDivId as insertBeforeId,
+							// so that in GDC frontend framework, plots that are launched from scRNAseq
+							// will be inserted before it. TODO: may insert after the scRNAseq plot instead???
```

Comment:
this.insertBeforeId or this.mainDivId?

Reply:
This div ID is really meant to identify the div itself (`mainDiv`), not really for what it will be used for later in an event listener (`insertBeforeId`). By renaming to `this.mainDivId`, it's clear what the element's role is - the fact that another element may be inserted before it, is not it's "persistent" role.

Reply:
The line 211 is still using this.insertBeforeId, that is not defined. That is what I meant

Reply:
I'll fix in Congyu's PR